File: app-pacientes-backend/src/main/java/pe/tintegro/apppacientesbackend/model/Parentesco.java
package pe.tintegro.apppacientesbackend.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Parentesco {

    private Integer idParentesco;
    private String parentesco;
    private Integer activo;

}

File: app-pacientes-frontend/src/app/data/types/Parentesco.ts
export class Parentesco {
  idParentesco: number;
  parentesco: string;

  constructor(
    idParentesco: number = 0,
    parentesco: string = ''
  ) {
    this.idParentesco = idParentesco;
    this.parentesco = parentesco;
  }
}
